pkg/webhook/validator.go

```diff
@@ -92,16 +93,27 @@ func (v *Validator) Handle(ctx context.Context, req types.Request) types.Respons
 		err = v.decoder.Decode(req, &pod)
 		podResult = validator.ValidatePod(v.Config, &pod.Spec)
 	} else {
-		var controller validator.Controller
-		switch req.AdmissionRequest.Kind.Kind {
-		case "Deployment":
+		var controller controllers.Interface
+		if yes := v.Config.CheckIfKindIsConfiguredForValidation(req.AdmissionRequest.Kind.Kind); !yes {
+			logrus.Warnf("Skipping, kind (%s) isn't something we are configured to scan", req.AdmissionRequest.Kind.Kind)
+			// FIXME: Should we be returning an OK response as skipped?
+			return admission.ErrorResponse(http.StatusBadRequest, err)
```

Comment:
Yes, I think for the webhook this should be an `OK` response for unsupported types. Otherwise (IIUC) if a new controller type is introduced, we'll reject it until Polaris gets updated to support it.

Reply:
Still seeing the `FIXME` here, and looks like it's still `StatusBadRequest`.

Reply:
I think maybe it was a bad browser cache on your side? I am not seeing it